src/SFA.DAS.EmployerIncentives/Infrastructure/Api/PassThroughApiClient.cs

```diff
@@ -0,0 +1,84 @@
+﻿using System;
+using System.Linq;
+using System.Net.Http;
+using System.Net.Http.Json;
+using System.Text.Json;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.WebUtilities;
+using SFA.DAS.EmployerIncentives.Interfaces;
+
+namespace SFA.DAS.EmployerIncentives.Infrastructure.Api
+{
+    public class PassThroughApiClient : IPassThroughApiClient
```

Comment:
Does this class give us anything above the RestApiClient?  The methods on here look like they could sit on there instead.

Reply:
Yes, it streamlines the json coming back from the inner api and just produces a json response. If we try to do this with the RestApiClient. We would have to add plumbing work simply to mirror the return types for each API call.

Because we are adding no value to the passthrough requests, it seems wrong to do anything but forward the response.